great_expectations/core/util.py

```diff
@@ -145,14 +142,14 @@ def in_jupyter_notebook():
         return False  # Probably standard Python interpreter
 
 
-def in_databricks() -> bool:
+def in_databricks(spark_session: pyspark.SparkSession) -> bool:
     """
     Tests whether we are in a Databricks environment.
 
     Returns:
         bool
     """
-    return "DATABRICKS_RUNTIME_VERSION" in os.environ  # noqa: TID251
+    return spark_session.sparkContext.appName == "Databricks Shell"
```

Comment:
Would it be safer to match here on containing `Databricks` instead of an exact match on `Databricks Shell`? Do we have confidence this naming won't change?